migrations/1724619807412-AddNameToReservationHallSeat.ts

```diff
@@ -0,0 +1,18 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class AddNameToReservationHallSeat1724619807412 implements MigrationInterface {
+    name = 'AddNameToReservationHallSeat1724619807412'
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`ALTER TABLE "reservation_hall_seat" RENAME COLUMN "reserved" TO "name"`);
```

Comment:
can we DROP COLUMN "reserved"  and then just ADD "name" character varying NOT NULL? do we need to rename here first?